index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Setup</title>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="container">
+    <section class="nav-bar">   
+      <a class="logo" href="">My Logo</a> 
+      <img class="union" src="/images/Union.png" alt="">   
+   </section> 
```

Comment:
Great job implementing the Toolbar! 📟 However, there are some things that you could do to improve it! 💯 
   - [ ] Let's use the `<header>` tag to wrap only the toolbar and not both sections, this will help to keep your code clean by keeping your sections well separated. 🛠️  Typically the header tag contains the site's title or logo (or both) and the main site navigation.  🤓 
   - [ ] It's important to wrap the hamburger button with a `<nav>` tag, indicating the browser that involves navigation. ⛵ This improves the semantics and accessibility of your project, by enabling screen readers to quickly identify how to navigate through your page. Please include it! 🛠️